django_webapps/scripts/db_population/populate_coordinators.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import os
+from dataclasses import dataclass
+
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "django_webapps.settings")
+django.setup()
+
+from django.db import IntegrityError, transaction  # noqa: E402
+from django.db.transaction import TransactionManagementError  # noqa: E402
+from django.utils import timezone  # noqa: E402
+
+from api.models.choices.status.project import CoordinatorProjectStatus  # noqa: E402
+from api.models.coordinator import Coordinator  # noqa: E402
+from django_webapps.scripts.db_population.base_populator import (  # noqa: E402
+    BasePersonCsvPopulator,
+    BasePersonEntityData,
+    CsvData,
+)
+from django_webapps.scripts.db_population.parsers import common_parsers  # noqa: E402
+from django_webapps.scripts.db_population.utils import (  # noqa: E402
+    get_args,
+    get_logger,
+    load_csv_data,
+)
+
+logger = get_logger("coordinators.log")
+LIST_OF_ADMINS_IDS = [22, 42]
+MAX_TID_WITH_NO_RESPONSE = 1300
+
+
+COLUMN_TO_ID = {
+    "tid": 0,
+    "name": 1,
+    "email": 2,
+    "tg": 3,
+    "timezone": 4,
+}
+
+
+@dataclass
+class CoordinatorData(BasePersonEntityData):
+    project_status: CoordinatorProjectStatus = CoordinatorProjectStatus.WORKING_OK
+    last_name: str = ""
+    is_validated: bool = False
+    is_admin: bool = False
+    utc_timedelta: datetime.timedelta = datetime.timedelta(hours=0)
+    status_since: datetime.datetime = timezone.now()
+
+
+class CoordinatorPopulator(BasePersonCsvPopulator):
+    entity_name: str = "coordinator"
+
+    def _pre_process_data(self, csv_data: CsvData) -> CsvData:
+        self.header[1] = "name"
+        self.header[4] = "timezone"
+        csv_data = csv_data[2:]
```

Comment:
just checking: is it consciously set to `2:`, not `1:`?

Reply:
It is correct since the first two columns are some trash data

Reply:
It is based on the csv from google sheet

Reply:
maybe a better approach would be to fix the google sheet, not sure

Reply:
nah :)